packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -252,7 +362,7 @@ export class TokenDetectionController extends StaticIntervalPollingController<
    * Starts a new polling interval.
    */
   async #startPolling(): Promise<void> {
-    if (this.#disabled) {
+    if (!this.isActive) {
```

Comment:
Are there any tests we need to add for this change?

Reply:
I'm okay with the tests being added in #3867, so I'm resolving this conversation.